src/utils.py

```diff
@@ -305,6 +307,32 @@ def get_object_combinations(
         yield list(choice)
 
 
+@functools.lru_cache(maxsize=None)
+def get_all_ground_atoms_for_predicate(predicate: Predicate,
```

Comment:
Would this be useful somehow in the side predicate planning stuff? (fine if not)

Reply:
resolved on slack (probably not)